Doc/library/subprocess.rst

```diff
@@ -33,9 +33,6 @@ The recommended approach to invoking subprocesses is to use the :func:`run`
 function for all use cases it can handle. For more advanced use cases, the
 underlying :class:`Popen` interface can be used directly.
 
-The :func:`run` function was added in Python 3.5; if you need to retain
```

Comment:
We normally keep the notes. on when a function was added.

There are many packages that need to maintain cross-version support even if we don't support the old versions.  This information is critical for them.

Reply:
The `.. versionadded:: 3.5` note for `subprocess.run()`, on line 103, is not touched by this PR. This PR proposes to remove an additional note, which seems to me to be overly prominent now that 3.4 has been end-of-life for a long time.